HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1415,25 +1396,25 @@ def self.get_default_heating_and_cooling_seasons(weather)
     heat_design_db = weather.design.HeatingDrybulb
 
     # create basis lists with zero for every month
-    cooling_season_temp_basis = Array.new(monthly_temps.length, 0.0)
-    heating_season_temp_basis = Array.new(monthly_temps.length, 0.0)
+    cooling_days_temp_basis = Array.new(monthly_temps.length, 0.0)
+    heating_days_temp_basis = Array.new(monthly_temps.length, 0.0)
```

Comment:
Why did all of the variables in this method change? The original names were correct.